packages/office-ui-fabric-react/src/components/OverflowSet/OverflowSet.types.ts

```diff
@@ -54,6 +54,13 @@ export interface IOverflowSetProps extends React.ClassAttributes<OverflowSetBase
    */
   vertical?: boolean;
 
+  /**
+   * Controls wether or not the DOM order is reversed.
```

Comment:
specify what is reversed. It's not quite the whole thing, just the placement of the overflow button vs the rest.

Reply:
updated to only reverse overflow position